src/components/impact/styledImpact.js

```diff
@@ -41,20 +105,48 @@ export const BoxText = styled.p`
     &:last-of-type {
         margin-bottom: 60px;
     }
+
+    @media (max-width: 425px) {
+        font-size: 14px;
+        line-height: 16px;
+    }
 `
 export const BoxDiamond = styled.img`
     width: 10px;
     height: 10px;
     margin: 20px 0;
+
+    @media (max-width: 425px) {
+        margin: 15px 0;
+    }
 `
 export const Grid = styled.div`
-    width: 83.33%;
+    width: 83.33%; 
     display: grid;
     margin: 60px auto 0 auto;
     grid-template-columns: repeat(6, 1fr);
     grid-template-rows: repeat(8, 1fr);
     border-radius: 20px;
     box-shadow: 0px 2px 10px rgba(0, 0, 0, 0.2);
+
+    @media (max-width: 1024px) {
+        width: 96.09%;     // 984/1024
+    }
+
+    @media (max-width: 425px) {
+        display: none;
```

Comment:
I think if you try overflow: scroll it is closer to what the design calls for. We would just need to style the scroll bar to match the design.